src/components/Footer.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+interface FooterProps {
+  todos: Todo[];
+  activeTodos: Todo[];
+  completedTodos: Todo[];
+  filter: 'all' | 'active' | 'completed';
+  setFilter: (filter: 'all' | 'active' | 'completed') => void;
+  handlClearAll: () => void;
+}
+
+export const Footer: React.FC<FooterProps> = ({
+  todos,
+  activeTodos,
+  completedTodos,
+  filter,
+  setFilter,
+  handlClearAll,
+}) => {
+  return (
+    <>
+      {todos.length > 0 && (
```

Comment:
Do this conditional render for the entire Footer component in the App.tsx instead of passing todos here